modules/migrating-between-capi-mapi.adoc

```diff
@@ -0,0 +1,153 @@
+:to-api-group: machine.openshift.io
+:origin-namespace: openshift-cluster-api
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="migrating-between-capi-mapi_{context}"]
+= Migrating a {from-api-name} resource to use the {to-api-name}
+
+You can migrate individual {from-api-name} objects to equivalent {to-api-name} objects.
+
+:FeatureName: Migrating a {from-api-name} resource to use the {to-api-name}
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster on a supported infrastructure type.
+
+ifdef::machine-to-cluster[]
+* You have enabled the use of the Cluster API.
+endif::[]
+
+* You have enabled the `MachineAPIMigration` feature gate in the `TechPreviewNoUpgrade` feature set.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Identify the {from-api-name} resource that you want to migrate to a {to-api-name} resource by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get <resource_kind> -n {origin-namespace}
+----
++
+--
+where `<resource_kind>` is one of the following values:
+
+`machine.{from-api-group}`:: The fully qualified name of the resource kind for a compute or control plane machine.
++
+[IMPORTANT]
+====
+The output of this command includes the control plane machines.
+Migrating a control plane machine in not supported.
+====
+
+`machineset.{from-api-group}`:: The fully qualified name of the resource kind for a compute machine set.
+--
+
+. Edit the resource specification by running the following command:
++
+[source,terminal]
+----
+$ oc edit <resource_name>
+----
++
+where `<resource_name>` is the fully qualified name of the {from-api-name} resource that you want to migrate to a {to-api-name} resource.
+
+. In the resource specification, update the value of the `spec.authoritativeAPI` field:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: <api_group>/<api_version> <1>
+kind: <resource_kind> <2>
+metadata:
+  name: <resource_name> <3>
+  [...]
+spec:
+  authoritativeAPI: {to-api-value} <4>
+  [...]
+status:
+  authoritativeAPI: {from-api-value} <5>
+  [...]
+----
+<1> The API group and version vary depending on the resource kind.
+<2> The resource kind varies depending on the resource kind.
+<3> The name of the resource that you want to migrate.
+<4> Specify the authoritative API that you want this resource to use.
+For example, to start migrating a {from-api-name} resource to the {to-api-name}, specify `{to-api-value}`.
+<5> The value for the current authoritative API.
+This value indicates which API currently manages this resource
+Do not change the value in this part of the specification.
+
+.Verification 
+
+* Check the status of the conversion by running the following command:
++
+[source,terminal]
+----
+$ oc -n {origin-namespace} get <resource_kind>/<resource_name> -o json | jq .status.authoritativeAPI
```

Comment:
Same as above, we can only check `.status.authoritativeAPI` in the `Machine API` resource, there is no `.status.authoritativeAPI` in the `Cluster API` resource.

Another, `{origin-namespace}` still shows `{origin-namespace}` in the doc https://drive.google.com/file/d/1Klv8p_ardBwNrhEJupLfPPS2m3lxEItv/view?usp=sharing

Reply:
I'll rework the conditions so that we can have a different verification step for CAPI. What command would they need to to a similar check for CAPI?